modules/tensor_mechanics/doc/content/source/kernels/MatVectorBodyForce.md

```diff
@@ -0,0 +1,19 @@
+# MatVectorBodyForce
```

Comment:
I was curious about the use of 'Mat' vs. 'Material', and I see 'Material' used much more commonly, especially in TM. I'd really like to see this class renamed to 'MaterialVectorBodyForce'. I think its meaning is much more clear.

Reply:
(moose) guillaume@ubuntu1:~/projects/moose/framework/src$ ll kernels/ | grep Mat
-rw-rw-r--  1 guillaume guillaume  349 Mar  4 10:44 ADMatDiffusionBase.C
-rw-rw-r--  1 guillaume guillaume  760 Mar  4 10:44 ADMatDiffusion.C
-rw-rw-r--  1 guillaume guillaume 3205 Mar  4 10:44 MatCoupledForce.C
-rw-rw-r--  1 guillaume guillaume  790 Mar  4 10:44 MatDiffusion.C
-rw-rw-r--  1 guillaume guillaume 2153 Mar  4 10:44 MaterialDerivativeRankFourTestKernel.C
-rw-rw-r--  1 guillaume guillaume 1808 Mar  4 10:44 MaterialDerivativeRankTwoTestKernel.C
-rw-rw-r--  1 guillaume guillaume 1387 Mar  4 10:44 MaterialDerivativeTestKernel.C
-rw-rw-r--  1 guillaume guillaume 1931 Jul  4 09:11 MaterialPropertyValue.C
-rw-rw-r--  1 guillaume guillaume 2686 Jul  4 09:11 MatReaction.C

Apart from those test kernels (why are they here??) Mat is more common in the framework. I guess this kernel is in tensor mechanics though

Reply:
I think 'Mat' isn't used at all in tensor mechanics. In general, we tend to try to minimize abbreviations to avoid ambiguity.